### DISCUSSION TITLE
How to create different databases depending on the testcase

### DISCUSSION THREAD

**Discussion by thomas-saigre** [feelpp/feelpp#1656] [Toolboxes]
When I ran the toolbox-mor (on python) on the case opus heat, all went well. But when I switched to another test case, the application found the databases saved in the directory so used it for the new case which led to an error.
According to @prudhomm there is a configuration somewhere to avoid it, it seems not to be working on my case

/cc @romainhild

**Comment by romainhild** [comment]
Indeed, that is because the database is written directly in the constructor, before, one application corresponded to one test case.
I have already thought of this and the solution is to add an option `toolboxmor.database` here: https://github.com/feelpp/feelpp/blob/70f6929bfdce6cdc48a8b54b4c077384b31e0c92/feelpp/feel/feelcrb/toolboxmor_impl.cpp#L9

And then call this option in the constructor, here replace `"ToolboxMor"` by `soption("toolboxmor.basename")` https://github.com/feelpp/feelpp/blob/70f6929bfdce6cdc48a8b54b4c077384b31e0c92/feelpp/feel/feelcrb/toolboxmor_impl.cpp#L43

One downside of this is to use the plugin for the online part. I'm not sure that it can find the database then.
